PhysicsTools/PatUtils/interface/SmearedJetProducerT.h

```diff
@@ -247,7 +247,7 @@ class SmearedJetProducerT : public edm::stream::EDProducer<> {
                     }
 
                     std::normal_distribution<> d(0, sigma);
-                    smearFactor = 1. + d(m_random_generator) / jet.energy();
+                    smearFactor = 1. + d(m_random_generator) / jet.pt();
```

Comment:
Is there a danger that `jet.pt()` could be zero so that there would be division by zero? Such jets might result from corrupted data or other odd cases. Line 237 also might have this problem.